setup.py

```diff
@@ -75,7 +75,9 @@ def run(self):
 s3 = ["boto3==1.9.115"]
 azure = ["azure-storage-blob==2.1.0"]
 oss = ["oss2==2.6.1"]
-ssh = ["paramiko[gssapi]>=2.5.0"]
+ssh = ["paramiko>=2.5.0"]
+# gssapi should not be included in all_remotes
```

Comment:
Maybe mention why, because following is supported only for pip package?

Reply:
Adjusted, thanks! :slightly_smiling_face: